bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/wizard/ImportIDFProjectWizard.java

```diff
@@ -48,7 +48,7 @@ protected IGenerator getGenerator()
 		locationStr = page.getLocation();
 		boolean canCopyIntoWorkspace = page.canCopyIntoWorkspace();
 
-		IDFProjectGenerator generator = new IDFProjectGenerator(null, new File(locationStr), canCopyIntoWorkspace);
+		IDFProjectGenerator generator = new IDFProjectGenerator(null, new File(locationStr), canCopyIntoWorkspace, "esp32");
```

Comment:
nit: we could extract "esp32" as DEFAULT_TARGET constant for better maintainability/readability